helix-view/src/editor.rs

```diff
@@ -571,7 +574,29 @@ impl Editor {
         self.status_msg = Some((error.into(), Severity::Error));
     }
 
+    pub fn unset_theme_preview(&mut self) {
+        if let Some(last_theme) = &self.last_theme.clone() {
```

Comment:
Why not use `self.last_theme.take()`? It can just be `None` after we take it, no?